### ISSUE TITLE
Successful packages to run cdvae

### ISSUE THREAD

**Issue by Saurav-Abid-Rahman** [txie-93/cdvae#52]
Try to install these packages if want to run cdvae. 
Package                       Version     Editable project location
----------------------------- ----------- -------------------------
absl-py                       1.4.0
aflow                         0.0.11
aiohttp                       3.8.5
aiosignal                     1.3.1
altair                        5.1.2
antlr4-python3-runtime        4.8
anyio                         4.0.0
appdirs                       1.4.4
APScheduler                   3.10.4
argon2-cffi                   23.1.0
argon2-cffi-bindings          21.2.0
arrow                         1.3.0
ase                           3.22.1
astor                         0.8.1
astroid                       3.0.0
asttokens                     2.4.0
async-lru                     2.0.4
async-timeout                 4.0.2
attrs                         23.1.0
autopep8                      2.0.4
Babel                         2.12.1
backcall                      0.2.0
backports.functools-lru-cache 1.6.5
backports.zoneinfo            0.2.1
base58                        2.1.1
beautifulsoup4                4.12.2
bleach                        6.0.0
blinker                       1.6.2
Brotli                        1.1.0
cached-property               1.5.2
cachetools                    5.3.1
cdvae                         0.0.1       /home/abid/cdvae
certifi                       2023.7.22
cffi                          1.15.1
cftime                        1.6.2
charset-normalizer            2.0.12
citrination-client            6.5.1
click                         8.1.6
colorama                      0.4.6
comm                          0.1.4
configparser                  6.0.0
contourpy                     1.1.0
cryptography                  41.0.4
cycler                        0.11.0
debugpy                       1.8.0
decorator                     5.1.1
defusedxml                    0.7.1
dill                          0.3.7
dnspython                     2.4.1
docker-pycreds                0.4.0
dpcpp-cpp-rt                  2023.2.0
dscribe                       2.1.0
entrypoints                   0.4
exceptiongroup                1.1.3
executing                     1.2.0
fair-research-login           0.3.1
fastjsonschema                2.18.1
filelock                      3.9.0
Flask                         3.0.0
fonttools                     4.41.1
fqdn                          1.5.1
frozenlist                    1.4.0
fsspec                        2023.6.0
future                        0.18.3
gitdb                         4.0.10
GitPython                     3.1.32
globus-nexus-client           0.4.1
globus-sdk                    3.28.0
gmpy2                         2.1.2
google-auth                   2.22.0
google-auth-oauthlib          1.0.0
googledrivedownloader         0.4
grpcio                        1.56.2
h11                           0.14.0
h2                            4.1.0
h5py                          3.9.0
hpack                         4.0.0
httpcore                      0.18.0
httplib2                      0.22.0
hydra-core                    1.1.0
hydra-joblib-launcher         1.1.5
hyperframe                    6.0.1
idna                          3.4
imageio                       2.31.1
importlib-metadata            6.8.0
importlib-resources           6.0.0
iniconfig                     2.0.0
intel-cmplr-lib-rt            2023.2.0
intel-cmplr-lic-rt            2023.2.0
intel-opencl-rt               2023.2.0
intel-openmp                  2023.2.0
ipykernel                     6.25.2
ipython                       8.12.2
ipywidgets                    8.1.1
isodate                       0.6.1
isoduration                   20.11.0
isort                         5.12.0
itsdangerous                  2.1.2
jedi                          0.19.1
Jinja2                        3.1.2
jmespath                      1.0.1
joblib                        1.1.0
json5                         0.9.14
jsonpointer                   2.4
jsonschema                    4.18.4
jsonschema-specifications     2023.7.1
jupyter_client                8.3.1
jupyter_core                  5.3.2
jupyter-events                0.7.0
jupyter-lsp                   2.2.0
jupyter_server                2.7.3
jupyter_server_terminals      0.4.4
jupyterlab                    4.0.6
jupyterlab-pygments           0.2.2
jupyterlab_server             2.25.0
jupyterlab-widgets            3.0.9
kiwisolver                    1.4.4
latexcodec                    2.0.1
lazy_loader                   0.3
lightning-utilities           0.9.0
llvmlite                      0.40.1
loguru                        0.7.2
Markdown                      3.4.4
MarkupSafe                    2.1.1
matminer                      0.7.3
matplotlib                    3.7.2
matplotlib-inline             0.1.6
mccabe                        0.7.0
mdf-forge                     0.8.0
mdf-toolbox                   0.5.8
mistune                       3.0.1
mkl                           2023.2.0
mkl-fft                       1.3.6
mkl-random                    1.2.2
mkl-service                   2.4.0
monty                         2023.5.8
mpmath                        1.2.1
multidict                     6.0.4
multiprocess                  0.70.15
munkres                       1.1.4
nbclient                      0.8.0
nbconvert                     7.8.0
nbformat                      5.9.2
nest-asyncio                  1.5.6
netCDF4                       1.6.4
networkx                      2.8.4
nglview                       3.0.8
notebook_shim                 0.2.3
numba                         0.57.1
numpy                         1.21.5
oauthlib                      3.2.2
omegaconf                     2.1.2
overrides                     7.4.0
p-tqdm                        1.3.3
packaging                     21.3
palettable                    3.3.3
pandas                        2.0.3
pandocfilters                 1.5.0
parso                         0.8.3
pathos                        0.3.1
pathtools                     0.1.2
patsy                         0.5.3
pexpect                       4.8.0
pickleshare                   0.7.5
Pillow                        10.0.0
Pint                          0.21.1
pip                           23.2.1
pkgutil_resolve_name          1.3.10
platformdirs                  3.11.0
plotly                        5.15.0
pluggy                        1.3.0
ply                           3.11
plyfile                       1.0.1
pooch                         1.7.0
pox                           0.3.3
ppft                          1.7.6.7
prometheus-client             0.17.1
promise                       2.3
prompt-toolkit                3.0.39
protobuf                      3.20.1
psutil                        5.9.0
ptyprocess                    0.7.0
pure-eval                     0.2.2
pyarrow                       13.0.0
pyasn1                        0.5.0
pyasn1-modules                0.3.0
pybind11                      2.11.1
pybind11-global               2.11.1
pybtex                        0.24.0
pycodestyle                   2.11.0
pycparser                     2.21
pydeck                        0.8.1b0
pyDeprecate                   0.3.0
Pygments                      2.16.1
PyJWT                         2.8.0
pylint                        3.0.0
pymatgen                      2022.4.26
pymongo                       4.4.1
pyOpenSSL                     23.2.0
pyparsing                     3.0.9
pypif                         2.1.1
PyQt5                         5.15.9
PyQt5-sip                     12.12.2
PySocks                       1.7.1
pytest                        7.4.2
python-dateutil               2.8.2
python-dotenv                 1.0.0
python-json-logger            2.0.7
python-louvain                0.16
pytorch-lightning             1.3.8
pytz                          2023.3
pyu2f                         0.1.5
PyWavelets                    1.4.1
PyYAML                        5.4.1
pyzmq                         25.1.1
rdflib                        6.3.2
referencing                   0.30.0
requests                      2.31.0
requests-mock                 1.11.0
requests-oauthlib             1.3.1
rfc3339-validator             0.1.4
rfc3986-validator             0.1.1
rpds-py                       0.9.2
rsa                           4.9
ruamel.yaml                   0.17.32
ruamel.yaml.clib              0.2.7
scikit-learn                  1.1.0
scipy                         1.7.3
seaborn                       0.13.0
Send2Trash                    1.8.2
sentry-sdk                    1.28.1
setproctitle                  1.3.2
setuptools                    59.5.0
shortuuid                     1.0.11
sip                           6.7.11
six                           1.16.0
SMACT                         2.2.1
smmap                         5.0.0
sniffio                       1.3.0
soupsieve                     2.5
sparse                        0.14.0
spglib                        2.0.2
stack-data                    0.6.2
statsmodels                   0.14.0
streamlit                     0.79.0
subprocess32                  3.5.4
sympy                         1.11.1
tabulate                      0.9.0
tbb                           2021.10.0
tenacity                      8.2.2
tensorboard                   2.14.1
tensorboard-data-server       0.7.1
termcolor                     2.3.0
terminado                     0.17.1
threadpoolctl                 3.1.0
tifffile                      2023.7.10
tinycss2                      1.2.1
toml                          0.10.2
tomli                         2.0.1
tomlkit                       0.12.1
toolz                         0.12.0
torch                         1.9.0+cu111
torch-cluster                 1.5.9
torch-geometric               1.7.2
torch-scatter                 2.0.9
torch-sparse                  0.6.12
torch-spline-conv             1.2.1
torchmetrics                  0.7.0
tornado                       6.3.3
tqdm                          4.65.0
traitlets                     5.11.2
types-python-dateutil         2.8.19.14
typing_extensions             4.7.1
typing-utils                  0.1.0
tzdata                        2023.3
tzlocal                       5.0.1
ujson                         5.8.0
uncertainties                 3.1.7
unicodedata2                  15.1.0
uri-template                  1.3.0
urllib3                       1.26.11
validators                    0.22.0
vtk                           9.2.6
wandb                         0.10.33
watchdog                      3.0.0
wcwidth                       0.2.8
webcolors                     1.13
webencodings                  0.5.1
websocket-client              1.6.3
Werkzeug                      2.3.6
wheel                         0.38.4
widgetsnbextension            4.0.9
wslink                        1.12.2
yacs                          0.1.8
yarl                          1.9.2
zipp                          3.16.2

Remember use the normal environment file that takes long time but eventually it works. You will face problem with version of torch, torch_sparse, torch_scatter, torch_cluster and torch_spline_conv. Try to install these mentioning their cuda compatibility. Like in my case it was cuda 11.1. As I installed torch 1.9.0 with cuda 11.1.
